examples/shaded-relief.js

```diff
@@ -126,12 +126,15 @@ function shade(inputs, data) {
 const elevation = new XYZ({
   url: 'https://{a-d}.tiles.mapbox.com/v3/aj.sf-dem/{z}/{x}/{y}.png',
   crossOrigin: 'anonymous',
+  maxZoom: 13,
 });
 
 const raster = new Raster({
   sources: [elevation],
   operationType: 'image',
   operation: shade,
+  // avoid interpolation artefacts by only rendering native elevation resolutions
+  resolutions: elevation.getTileGrid().getResolutions(),
```

Comment:
Do you think the first source's resolutions would make a good default?

Reply:
Maybe that would be a breaking change.  But also maybe something to consider later.

Reply:
I was also thinking about using the first source's resolutions as default, but then thought it might complicate things for the user when multiple sources are involved.